src/main/java/edu/mbrooks/advancedjava/impl/Person.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * Program Name:    Person.java
+ * Author:          Michael Brooks
+ * Version:         Version 1.0
+ * Course:          Advanced Java Programming, Summer 2018
+ * Copyright:       (C) 2018 Michael A. Brooks
+ * Description:     Implementation of the person interface
+ */
+package edu.mbrooks.advancedjava.impl;
+
+import edu.mbrooks.advancedjava.interfaces.Iperson;
+
+import java.util.GregorianCalendar;
+
+/**
+  * @author michaelbrooks
+  * class implements a person
+ */
+public class Person implements Iperson {
+
+    private String name;
+    private String address;
+    private String ssn;
+    private java.util.GregorianCalendar dob;
+
+    /**
+     * Default constructor sets the name to an empty string
```

Comment:
technically this is not a default constructor. The default constructor takes no arguments. If a class has no constructors at all, the default constructor is automatically generated.